server/src/deployment-center/github/frameworks.json

```
@@ -52,6 +52,28 @@
       "outputLocation": "build"
     }
   },
+  {
+    "id": "react webpack",
+    "name": "React Webpack",
+    "kind": "app",
+    "overrides": [
+      "react"
+    ],
+    "packages": [
+      "react",
+      "react-dom",
```

Remove trailing comma (illegal in JSON).
```suggestion
      "react-dom"
```